src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -17,6 +17,8 @@ type state = {
   mutable packages : Info.t Version.Map.t Name.Map.t;
   mutable stats : Statistics.t option;
   mutable featured : t list option;
+  mutable build_check_digest : Digest.t option;
+  mutable build_check : (string * Check.status) list String.Map.t;
```

Comment:
Can we expose this in the GraphQL API also?

Reply:
![Screenshot_2023-06-02_11-24-24](https://github.com/ocaml/ocaml.org/assets/1917592/924ad4ce-14b0-44f7-ac51-122c51676877)

Reply:
![Screenshot_2023-06-02_13-10-19](https://github.com/ocaml/ocaml.org/assets/1917592/03d3ef33-27b3-4112-9e33-c14b29c0dbe0)